packages/fx-core/src/component/feature/cicd/cicd.ts

```diff
@@ -56,7 +54,7 @@ export class CICD {
   ): Promise<Result<undefined, FxError>> {
     const cicdImpl: CICDImpl = new CICDImpl();
     const envName = inputs.env || inputs[questionNames.Environment];
-    const res = await cicdImpl.addCICDWorkflows(context, inputs, envName);
+    const res = await cicdImpl.addCICDWorkflows(context, inputs, envName, context.envInfo);
```

Comment:
Do we still need to add `context.envInfo` as parameter when `context` is already passed as the first one?

Reply:
Fixed.